subprojects/tracy.wrap

```diff
@@ -0,0 +1,5 @@
+[wrap-git]
```

Comment:
What is the purpose of this file? I don't know what a .wrap file is.

Reply:
It's a way to manage dependencies with meson: https://mesonbuild.com/Wrap-dependency-system-manual.html

Reply:
Ok, so this is something Meson supports as-is? Do we know if the Meson Proton builds with supports this?

Reply:
I think Proton uses the soldier container, which is based on debian 10 (buster). So unless soldier has backported a new version, vkd3d-proton is stuck with meson 0.49.

Reply:
Tried building this PR in the Proton docker and it builds fine with default setup, so I think it's safe.